client/src/pages/reports/AttendeesTable.js

```diff
@@ -36,8 +38,9 @@ const TableHeader = ({ showDelete, hide }) => (
       </th>
       <th className="col-xs-3">{!hide && "Name"}</th>
       <th className="col-xs-3">{!hide && "Position"}</th>
-      <th className="col-xs-2">{!hide && "Location"}</th>
+      <th className="col-xs-1">{!hide && "Location"}</th>
       <th className="col-xs-2">{!hide && "Organization"}</th>
+      <th className="col-xs-1" />
```

Comment:
Shouldn't this header have some text in it?

Reply:
I don't think it is necessary because we are displaying conflict icon in corresponding cell and it is not always displayed. When it is displayed, icon has a tooltip explaining its purpose. When not displayed there is no point of showing a title like 'Conflicts'.